src/player.js

```diff
@@ -427,6 +427,7 @@ export default class Player extends FakeEventTarget {
     this._appendDomElements();
     this._externalCaptionsHandler = new ExternalCaptionsHandler(this);
     this._fullscreenController = new FullscreenController(this);
+    this._handleAdsWithMSE();
```

Comment:
need to add it to `_attachMedia` cause on change media flow calls reset that erases the event listeners

Reply:
Ok, I missed it